README.md

```
@@ -97,13 +98,53 @@ In **Expert mode**, Ivan splits into two minds. The **Implementer** writes the c
 
+## 🔁 Loop Mode: keep going until it's actually *good*
+
+Expert mode answers *"is this correct?"* Loop mode asks the next question — *"is this good?"* — and keeps iterating until the answer is yes.
+
+`--mode loop` runs the **full expert collaboration first** (so improvements are built on a correct baseline), then adds a fourth phase: a **product-minded reviewer** looks at your original request and the implemented diff and hunts for what would make it genuinely better — in **features** and in **UX**. In-scope improvements get built; larger ideas get surfaced to the PR instead of quietly ballooning it.
+
+```bash
+ivan --mode loop "Add a CLI command to export analytics as CSV"
+```
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)